.github/workflows/ci.yml

```
@@ -161,7 +161,12 @@ jobs:
     steps:
       - uses: taiki-e/checkout-action@v1
       - name: Install Rust
-        run: rustup toolchain add nightly --no-self-update && rustup default nightly
+        run: rustup toolchain add stable --no-self-update --profile minimal && rustup default stable
```

```suggestion
        run: rustup update stable --no-self-update
```

stable is already the default toolchain, and minimal is the default profile AFAIK